### DISCUSSION TITLE
PSA: Functional Tests CI workflows are failing randomly, most possibly due to GitHub Actions platform error

### DISCUSSION THREAD

**Discussion by Piedone** [OrchardCMS/OrchardCore#16194] [General]
You may see the Functional Tests CI workflows randomly fail with "The hosted runner encountered an error while running your job. (Error Type: Failure)." errors, like [here](https://github.com/OrchardCMS/OrchardCore/actions/runs/9293755080). This is most possibly a platform issue, reported by others too: https://github.com/actions/runner-images/issues/9959. I expect this to be resolved by GitHub soon, but until then, be aware that these workflows can fail even if you didn't mess up anything.

Rerunning the workflow may get you to a point where a different job fails (e.g. "Functional Tests - CMS SqlServer 2019" instead of "Functional Tests - CMS MySql"). If with the same code you see all five jobs succeed eventually, then the PR may be merged, because then the Functional Tests are passing.

This is foremost for people with open recent PRs @gvkries @hishamco @MikeAlhayek @Skrypt.

**Comment by hishamco** [comment]
You read my mind @Piedone I just noticed that the last PR skipped one of the functional testing git workflow (MySQL)

**Comment by Piedone** [comment]
This seems to be fixed now.
